requirements.txt

```diff
@@ -7,3 +7,4 @@ pytest>=3.3.1
 task-queue>=0.4.1
 -e git+https://github.com/seung-lab/tqdm.git#egg=tqdm
 oauth2client==3.0.0
+scikit-image>=0.14.1
```

Comment:
Might be a good idea to have a separate requirements file for chunkflow only stuff. People have complained to me about the number of dependencies igneous has.

Reply:
I agree. igneous have too much dependency. I had some trouble dealing with package version conflicts which I am not using in chunkflow. 
Actually, I am working on separating chunkflow as standalone package.

Reply:
In that case, we can just close this PR?

Reply:
If that's what you'd like to do.